app/api/tasks_api.rb

```diff
@@ -24,19 +24,18 @@ class TasksApi < Grape::API
         error!({ error: 'You do not have permission to read these task details' }, 403)
       end
 
+      # TODO (stream) Do we need tutorial here?
```

Comment:
@macite Any suggestions?

Reply:
Yes, we need tutorial_id here.
We should add stream_id as well.